README.md

```diff
@@ -132,37 +132,40 @@ Usage:
-  -wcc, --with-callconv <with-callconv>                      A calling convention to be used for the given declaration during binding generation. [default: ]
-  -wc, --with-class <with-class>                             A class to be used for the given remapped constant or function declaration name during binding generation. [default: ]
-  -wlb, --with-librarypath <with-librarypath>                A library path to be used for the given declaration during binding generation. [default: ]
-  -wn, --with-namespace <with-namespace>                     A namespace to be used for the given remapped declaration name during binding generation. [default: ]
-  -wsle, --with-setlasterror <with-setlasterror>             Add the SetLastError=true modifier to a given DllImport or UnmanagedFunctionPointer. [default: ]
-  -wts, --with-transparent-struct <with-transparent-struct>  A remapped type name to be treated as a transparent wrapper during binding generation. [default: ]
-  -wt, --with-type <with-type>                               A type to be used for the given enum declaration during binding generation. [default: ]
-  -wu, --with-using <with-using>                             A using directive to be included for the given remapped declaration name during binding generation. [default: ]
+  -was, --with-access-specifier <with-access-specifier>      An access specifier to be used with the given qualified or remapped declaration name during binding generation. []
+  -wa, --with-attribute <with-attribute>                     An attribute to be added to the given remapped declaration name during binding generation. []
+  -wcc, --with-callconv <with-callconv>                      A calling convention to be used for the given declaration during binding generation. []
+  -wc, --with-class <with-class>                             A class to be used for the given remapped constant or function declaration name during binding generation. []
+  -wg, --with-guid <with-guid>                               A GUID to be used for the given declaration during binding generation. []
+  -wlb, --with-librarypath <with-librarypath>                A library path to be used for the given declaration during binding generation. []
+  -wmi, --with-manual-import <with-manual-import>            A remapped function name to be treated as a manual import during binding generation. []
+  -wn, --with-namespace <with-namespace>                     A namespace to be used for the given remapped declaration name during binding generation. []
+  -wsle, --with-setlasterror <with-setlasterror>             Add the SetLastError=true modifier to a given DllImport or UnmanagedFunctionPointer. []
+  -wsgct, --with-suppressgctransition <with-suppressgctransition>  Add the SuppressGCTransition calling convention to a given DllImport or UnmanagedFunctionPointer. []
```

Comment:
Can also normalize spaces for the others if preferred. It does start to cut off a lot quicker though.